samples/SimpleWebApp.Tests/EndToEndTests.cs

```diff
@@ -12,12 +12,17 @@ public async Task Search()
     {
         var query = """
             {
-              search (forString:"") {
+              search (forString: "", first: 100) {
                 items {
                   __typename
                   ... on Book {
                     id
                     title
+                    authors {
```

Comment:
Enhanced test for SampleWebApp here, as previously the test did not test the `[Inject]` bit of code

Reply:
The mutation added in PR #252 does test `[Inject]` actually.